.gitlab-ci.yml

```diff
@@ -1,283 +1,4 @@
-
-pup4-lint:
-  <<: *pup_4
-  <<: *lint_tests
-
-pup5-lint:
-  <<: *pup_5
-  <<: *lint_tests
-
-pup6-lint:
-  <<: *pup_6
-  <<: *lint_tests
-
-# Unit Tests
-#-----------------------------------------------------------------------
-
-pup5-unit:
-  <<: *pup_5
-  <<: *unit_tests
-
-pup5.5.7-unit:
-  <<: *pup_5_5_7
-  <<: *unit_tests
-
-pup4.10-unit:
-  <<: *pup_4_10
-  <<: *unit_tests
-
-pup6-unit:
-  <<: *pup_6
-  <<: *unit_tests
-
-# Acceptance tests
-# ==============================================================================
-
-pup4.10:
-  <<: *acceptance_base
-  <<: *pup_4_10
-  script:
-    - 'bundle exec rake beaker:suites[default]'
-
-pup4.10-fips:
-  <<: *acceptance_base
-  <<: *pup_4_10
-  <<: *only_with_SIMP_FULL_MATRIX
-  script:
-    - 'BEAKER_fips=yes bundle exec rake beaker:suites[default]'
-
-pup4.10-prelink_fact:
-  <<: *acceptance_base
-  <<: *pup_4_10
-  script:
-    - 'BEAKER_fips=yes bundle exec rake beaker:suites[prelink_fact]'
-
-pup4.10-prelink_fact-fips:
-  <<: *acceptance_base
-  <<: *pup_4_10
-  <<: *only_with_SIMP_FULL_MATRIX
-  script:
-    - 'BEAKER_fips=yes bundle exec rake beaker:suites[prelink_fact]'
-
-pup4.10-ipa_fact:
-  <<: *acceptance_base
-  <<: *pup_4_10
-  <<: *only_with_SIMP_FULL_MATRIX
-  script:
-    - 'bundle exec rake beaker:suites[ipa_fact]'
-
-pup5.5.7:
-  <<: *acceptance_base
-  <<: *pup_5_5_7
-  script:
-    - 'bundle exec rake beaker:suites[default]'
-
-pup5.5.7-fips:
-  <<: *acceptance_base
-  <<: *pup_5_5_7
-  script:
-    - 'BEAKER_fips=yes bundle exec rake beaker:suites[default]'
-
-pup5.5.7-prelink_fact:
-  <<: *acceptance_base
-  <<: *pup_5_5_7
-  script:
-    - 'bundle exec rake beaker:suites[prelink_fact]'
-
-pup5.5.7-prelink_fact-fips:
-  <<: *acceptance_base
-  <<: *pup_5_5_7
-  script:
-    - 'BEAKER_fips=yes bundle exec rake beaker:suites[prelink_fact]'
-
-pup5.5.7-ipa_fact:
-  <<: *acceptance_base
-  <<: *pup_5_5_7
-  script:
-    - 'bundle exec rake beaker:suites[ipa_fact]'
-
-pup5.5.7-oel:
-  <<: *acceptance_base
-  <<: *pup_5_5_7
-  script:
-    - 'bundle exec rake beaker:suites[default,oel]'
-
-pup5.5.7-oel-fips:
-  <<: *acceptance_base
-  <<: *pup_5_5_7
-  <<: *only_with_SIMP_FULL_MATRIX
-  script:
-    - 'BEAKER_fips=yes bundle exec rake beaker:suites[default,oel]'
-
-pup5.5.7-oel-prelink_fact:
-  <<: *acceptance_base
-  <<: *pup_5_5_7
-  script:
-    - 'bundle exec rake beaker:suites[prelink_fact,oel]'
-
-pup5.5.7-oel-prelink_fact-fips:
-  <<: *acceptance_base
-  <<: *pup_5_5_7
-  <<: *only_with_SIMP_FULL_MATRIX
-  script:
-    - 'BEAKER_fips=yes bundle exec rake beaker:suites[prelink_fact,oel]'
-
-# The default node set for the ipa_fact suite includes OEL servers
-#pup5.5.7-oel-ipa_fact:
-#  <<: *acceptance_base
-#  <<: *pup_5_5_7
-#  <<: *only_with_SIMP_FULL_MATRIX
-#  script:
-#    - 'bundle exec rake beaker:suites[ipa_fact,oel]'
+include:
+  - project: 'simp/remote-gitlab-ci'
+    file:    '/gitlab-templates/.basic.gitlab-ci.yml'
```

Comment:
(Following up on today's discussion:) Do not merge this yet.

This particular use of `include` and the current architecture of [simp/remote-gitlab-ci](https://github.com/simp/remote-gitlab-ci) will cause increasingly problematic RELENG-related issues over time.  This is a design issue, and it must be addressed _before_ introducing a centralized `include` strategy across all our repositories, or we may irretrievably lose the ability to run meaningful pipelines across different projects over time.

I'll follow up in more detail in issue [SIMP-5650](https://simp-project.atlassian.net/browse/SIMP-5650).